docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -1101,6 +1105,33 @@ public class TestClientRequestFilter implements ResteasyReactiveClientRequestFil
 }
 ----
 
+== Customizing the ObjectMapper in REST Client Reactive Jackson
+
+The REST Client Reactive supports adding a custom ObjectMapper to be used only the Client using the annotation `@ClientObjectMapper`. 
+
+A simple example is to provide a custom ObjectMapper to the REST Client Reactive Jackson extension by doing:
+
+[source, java]
+----
+@Path("/extensions")
+@RegisterRestClient
+public interface ExtensionsService {
+
+    @GET
+    Set<Extension> getById(@QueryParam("id") String id);
+
+    @ClientObjectMapper <1>
+    static ObjectMapper objectMapper(ObjectMapper defaultObjectMapper) { <2>
+        return defaultObjectMapper.copy()
```

Comment:
We should mention here that users should `NEVER` change the `defaultObjectMapper` but instead always use `copy` if they pan to inherit the default settings. 
The same goes for the Javadoc of `@ClientObjectMapper`

Reply:
Added